packages/themes/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "travix-themes",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Travix International",
+  "license": "private"
```

Comment:
we need to specify the licence and also we need licence file that describes the permissions.
Here we can select the desired options and generate the static text for that `LICENCE` file and later on we can use embedded version on the website where we are going to present the themes (the style guide)

Reply:
further more i think this one is enough - https://creativecommons.org/licenses/by-sa/4.0/